src/main/webapp/i18n/zh-cn/administratorRole.json

```diff
@@ -2,10 +2,10 @@
   "clubmanagementApp": {
     "AdministratorRole": {
       "null": "",
-      "CC_HEAD": "CC_HEAD",
-      "VICE_CC_HEAD": "VICE_CC_HEAD",
-      "SECRETARY": "SECRETARY",
-      "TEASURER": "TEASURER"
+      "CC_HEAD": "CC HEAD",
+      "VICE_CC_HEAD": "副 CC HEAD",
+      "SECRETARY": "秘书",
+      "TEASURER": "财政"
```

Comment:
TYPO for the "TEASURER" as well

Reply:
Resolved by https://github.com/Five-Fishes/Club-Management-React-Client/pull/156/commits/755795e36a8942cbd7b3ef2cc7ee04047c26d74b